requirements.txt

```diff
@@ -0,0 +1,13 @@
+#### begin pypi dependencies ####

```

Comment:
Both Redis and pyTest are not PTF dependencies. Also, they are key components of the framework. Due to this, they should be installed explicitly.

Reply:
There are no mentions that pytest or redis are ptf dependencies.
requirements.txt file is a pythonic way to group/freeze dependencies.
What do you mean by explicitly?